app/controllers/avo/search_controller.rb

```diff
@@ -120,12 +121,18 @@ def apply_search_metadata(models, avo_resource)
       models.map do |model|
         resource = avo_resource.dup.hydrate(model: model).hydrate_fields(model: model)
 
+        if resource.search_result_url
+          url = resource.search_result_url.call(model, main_app, params)
```

Comment:
I had to pass `main_app` to be able to use url helpers in the lambda.
I had to pass `params` to be able to check if the search is global or not

Reply:
If we use the Evaluation Host we will be able to delegate `main_app` to the `view_context`.
See [this](https://github.com/avo-hq/avo/blob/main/lib/avo/fields/base_field.rb#L19-L22).

Reply:
I've changed to use the search host, but I think we still need to pass `model` since the evaluation host does not seem to have the resource model instance. What do you think?